### PR TITLE
Sanitised Form Builder dev namespace configuration with Mitch's help.

### PR DESCRIPTION
**Pull request by solidgoldpig**
And cloned staging from updated dev.

### CODE REVIEW

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-staging/resources/main.tf [hunk 1]
```diff
@@ -5,32 +5,3 @@ terraform {
 provider "aws" {
   region = "eu-west-1"
 }
-
-module "rds-instance" {
```
Comment (alkar) [inline namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-staging/resources/main.tf]: I think you might have accidentally removed this RDS instance?

[the remark this answers is not in the document]
Comment (solidgoldpig) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-staging/resources/main.tf]: Yep - this is to reflect the change in the dev namespace

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-staging/resources/publisher.tf [hunk 2]
```diff
@@ -0,0 +1,119 @@
+##################################################
+# Publisher RDS
+
+module "publisher-rds-instance" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=2.1"
```
[the remark this answers is not in the document]
Comment (solidgoldpig) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-staging/resources/publisher.tf]: Done

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-staging/resources/publisher.tf [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  data {
+    primary_endpoint_address = "${module.publisher-elasticache.primary_endpoint_address}"
+    auth_token               = "${module.publisher-elasticache.auth_token}"
+  }
+}
+
+########################################################
+
+# Publisher ECR Repos
+module "ecr-repo-fb-publisher-base" {
```
[the remark this answers is not in the document]
Comment (solidgoldpig) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-staging/resources/publisher.tf]: Done

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-staging/resources/user-datastore.tf [hunk 4]
```diff
@@ -0,0 +1,49 @@
+##################################################
+# Publisher RDS
+module "user-datastore-rds-instance" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=2.1"
```
[the remark this answers is not in the document]
Comment (solidgoldpig) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-staging/resources/user-datastore.tf]: Done

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-staging/resources/publisher.tf (hunk 3 above)
[the remark this answers is not in the document]
Comment (solidgoldpig) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-staging/resources/publisher.tf]: Or rather not done.

Repo modules all now moved to dev/resources/repos.tf

**Pull request merged**